src/qt/rpcconsole.cpp

```diff
@@ -462,13 +462,19 @@ void RPCConsole::clear()
     }
 
     // Set default style sheet
+    QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont());
+    // Try to make fixed font adequately large on different OS
+    QString ptSize = QString("%1pt").arg(QFontInfo(QFont()).pointSize() * 8.5 / 9);
```

Comment:
@luke-jr  `fixedFontInfo.pointSize()` is `12`, so way too large

@jonasschnelli `QFontInfo(QFont()).pointSize()` is `9`, often. So we get a `8.5pt` in this case instead of a longer float.